### PR TITLE
Add scripts to recreate the database

### PR DESCRIPTION
**Pull request by janosbrodbeck**
Adds a PowerShell and bash script to recreate the database and its volume. Just a convenience script

### CODE REVIEW

**Comment by janosbrodbeck** [comment]
Nvm I need to delete the volume with the docker compose file since the name is currently determined by the foldername.

File: scripts/recreate_database.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+path="$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}"; )" &> /dev/null && pwd 2> /dev/null; )";
```
Comment (dolevi-tub) [inline scripts/recreate_database.sh:3]: Please add a brief explanatory comment to this statement.
People with very limited scripting knowledge (such as I am!) take a long time to assess this.

File: scripts/recreate_database.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+# cd into root path of the backend project
+cd "${path}
```
Comment (dolevi-tub) [inline scripts/recreate_database.sh:6]:
```suggestion
cd "${path}"
```

**Review by dolevi-tub — CHANGES_REQUESTED** [review]

File: scripts/recreate_database.sh (hunk 2 above)
Comment (janosbrodbeck) [reply scripts/recreate_database.sh]: Fixed

File: scripts/recreate_database.sh (hunk 1 above)
Comment (janosbrodbeck) [reply scripts/recreate_database.sh:5]: Added a brief note to it. Is it sufficient?

**Review by dolevi-tub — APPROVED** [review]
Ready to merge

**Pull request merged**